test/integration/abuseReportTest.ts

```diff
@@ -58,85 +50,73 @@ describe("Test: Reporting abuse", async () => {
         let badEvent2Comment = `COMMENT: ${Math.random()}`;
 
         console.log("Test: Reporting abuse - send reports");
-        let reportsToFind = []
-
-        // Time to report, first without a comment, then with one.
-        try {
-            await goodUser.doRequest("POST", `/_matrix/client/r0/rooms/${encodeURIComponent(roomId)}/report/${encodeURIComponent(badEventId)}`);
-            reportsToFind.push({
+        let reportsToFind = [
+            {
                 reporterId: goodUserId,
                 accusedId: badUserId,
                 eventId: badEventId,
                 text: badText,
                 comment: null,
-            });
-        } catch (e) {
-            console.error("Could not send first report", e.body || e);
-            throw e;
-        }
-
-        try {
-            await goodUser.doRequest("POST", `/_matrix/client/r0/rooms/${encodeURIComponent(roomId)}/report/${encodeURIComponent(badEventId2)}`, "", {
-                reason: badEvent2Comment
-            });
-            reportsToFind.push({
+            },
+            {
                 reporterId: goodUserId,
                 accusedId: badUserId,
                 eventId: badEventId2,
                 text: badText2,
                 comment: badEvent2Comment,
-            });
-        } catch (e) {
-            console.error("Could not send second report", e.body || e);
-            throw e;
-        }
-
-        try {
-            await goodUser.doRequest("POST", `/_matrix/client/r0/rooms/${encodeURIComponent(roomId)}/report/${encodeURIComponent(badEventId3)}`, "");
-            reportsToFind.push({
+            },
+            {
                 reporterId: goodUserId,
                 accusedId: badUserId,
                 eventId: badEventId3,
                 text: badText3,
                 comment: null,
-            });
-        } catch (e) {
-            console.error("Could not send third report", e.body || e);
-            throw e;
-        }
-
-        try {
-            await goodUser.doRequest("POST", `/_matrix/client/r0/rooms/${encodeURIComponent(roomId)}/report/${encodeURIComponent(badEventId4)}`, "");
-            reportsToFind.push({
+            },
+            {
                 reporterId: goodUserId,
                 accusedId: badUserId,
                 eventId: badEventId4,
                 text: null,
                 textPrefix: badText4.substring(0, 256),
                 comment: null,
-            });
-        } catch (e) {
-            console.error("Could not send fourth report", e.body || e);
-            throw e;
-        }
-
-        try {
-            await goodUser.doRequest("POST", `/_matrix/client/r0/rooms/${encodeURIComponent(roomId)}/report/${encodeURIComponent(badEventId5)}`, "");
-            reportsToFind.push({
+            },
+            {
```

Comment:
`// Report containing long that should be truncated by the report manager because it contains too many newlines.`